src/XRP/issued-currency-client.ts

```diff
@@ -674,8 +674,8 @@ export default class IssuedCurrencyClient {
    */
   public async redeemIssuedCurrency(
     sender: Wallet,
```

Comment:
I'm going to make an Asana task for this since I agree, but figure it'll be best to do all at once and once everything's implemented so we can guarantee consistency.  https://app.asana.com/0/1190819649053561/1199151116842789/f